addons/event/static/src/scss/event.scss

```diff
@@ -32,13 +34,24 @@
         }
         .o_event_bottom_right {
             width: 60px;
-            right: 8px;
-            bottom: 8px;
+            height: 20px;
+            margin-top: -28px;
```

Comment:
@awa-odoo 
The kanban card has been improve in another task ;)
So, it's not useful to spend too munch time on this one I think as I will need to rebase in any way

Reply:
Then why don't we merge both those tasks?
Or wait until the other one is merged before continuing this one?

I'm not a big fan of that kind of thinking.
Let's say the other task is blocked for whatever reason, we end up with crappy css in master.

Reply:
@awa-odoo If you prefer, we can merge both tasks

Reply:
Well that's not for me to decide.

It depends on a lot of things:
- Is the other one starred as well?
- Is it further in the pipeline / going to be merged soon ?
- Does it imply big specifications that will change this PR implementation?
- Can we split a small "preparation" commit in the other PR that can be put in both PRs to speed up things?
- What does the PO think about it?
- ...

You obviously have more context than me on this so I let you choose the best solution.
Either way, leaving un-reviewed code because "we are going to fix it later" is never a very good choice, so I would rather avoid it if we can.

Reply:
@awa-odoo 
The other task is also starer, but it's in testing BE and it's green, I think the best it to not merge the tasks, (not the same PO, not the same position in the pipeline...)

We will not have "weird CSS" because the other task is more advanced in the pipe than this one, I think we should merge the "kanban style task" as soon as possible, to rebase this one

Just my opinion